distribution/docs/src/main/resources/_contents/_plugins/metacardingest-network-contents.adoc

```diff
@@ -0,0 +1,75 @@
+==== Metacard Ingest Network Plugin
```

Comment:
I don't think it's as important for it to match the module. (I'm fine with being overruled on this though)